lib/model/game_state.dart

```diff
@@ -79,4 +79,9 @@ class GameState with ChangeNotifier {
 
     notifyListeners();
   }
+
+  void endGame() {
+    // TODO - other tidying up?
```

Comment:
In line with other suggestion, could set the result here. Don't think we need other tidying up as starting a new game should just create a brand new state (rather than trying to reset this one)